src/main/scala/mesosphere/marathon/api/akkahttp/v2/GroupsController.scala

```diff
@@ -20,8 +29,19 @@ class GroupsController(electionService: ElectionService, infoService: GroupInfoS
     val authorizer: Authorizer
 ) extends Controller {
   import Directives._
+  import mesosphere.marathon.api.akkahttp.EntityMarshallers._
+  import mesosphere.marathon.api.v2.json.Formats._
 
-  def groupDetail(groupId: PathId): Route = ???
+  def groupDetail(groupId: PathId)(implicit identity: Identity): Route = {
+    extractEmbeds {
+      case (appEmbed, groupEmbed) =>
+        onSuccess(infoService.selectGroup(groupId, authorizationSelectors, appEmbed, groupEmbed)) {
+          case Some(info) => complete(Json.toJson(info))
+          case None if groupId.isRoot => complete(Json.toJson(GroupInfo.empty))
```

Comment:
Same here. I thought it would be `raml.GroupInfo.empty` or something similar.

Reply:
for GroupInfo RAML, AppInfo RAML is needed - we don't have that and I think it should not be part of this diff

Reply:
You are right. Could you create a JIRA as I mentioned above?

Reply:
https://jira.mesosphere.com/browse/MARATHON-7900